user-management/src/main/elm/sources/View.elm

```diff
@@ -8,10 +8,11 @@ module View exposing (..)
 import ApiCalls exposing (deleteUser)
 
 import DataTypes exposing (PanelMode(..), Model, Msg(..), RoleConf, Roles, StateInput(..), User, Username, Users, UsersConf)
+import Debug exposing (log)
 import Dict exposing (keys)
-import Html exposing (Html, a, br, button, div, h2, h3, h4, h5, i, input, p, span, text)
-import Html.Attributes exposing (attribute, class, disabled, href, id, placeholder, required, type_, value)
-import Html.Events exposing (onClick, onInput)
+import Html exposing (Html, a, br, button, div, h2, h3, h4, h5, i, input, label, p, span, text)
+import Html.Attributes exposing (attribute, checked, class, disabled, for, href, id, name, placeholder, required, type_, value)
```

Comment:
use ```exposing(..)``` for HTML and Html.Attributes it will import all function of both module and will really ease your life